samza-core/src/main/java/org/apache/samza/zk/ZkKeyBuilder.java

```diff
@@ -88,5 +88,7 @@ public String getJobModelPath(String jobModelVersion) {
     return String.format("%s/%s", getJobModelPathPrefix(), jobModelVersion);
   }
 
-
+  public String getJobModelVersionBarrierPrefix() {
+    return String.format("/%s/versionBarriers", pathPrefix);
```

Comment:
Alright.  I think I was misled by the function getJobModelPath(), that one has a "/" at the beginning. I think we should keep them in consistency for the convention about path prefix.